.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: pass-ember-adapter Continuous Integration
+on: [pull_request]
+
+concurrency:
+  group: ci-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  print-workflow-description:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "This is a CI build of branch ${{ github.ref }} in repository ${{ github.repository }}"
+      - run: echo "This job was triggered by a ${{ github.event_name }} event and is running on a ${{ runner.os }} server"
+
+  call-tests-unit:
+    name: Run Unit Tests
+    uses: eclipse-pass/pass-ember-adapter/.github/workflows/tests-unit.yml@171-gh-actions-test
```

Comment:
The references to `171-gh-actions-test` should be updated to `main`.

Reply:
I undid that extra commit